CHANGELOG.rst

```diff
@@ -37,3 +37,7 @@ packages formula
 0.0.9 (2018-10-03)
 
 - Add support for plain files using arcihve
+
+0.1.0 (2019-03-10)
+
+- Add npm support
```

Comment:
Could we get this file reverse chronological before it gets out of hand?  For consistency with our other changelogs and particularly for the automated version we hope to bring in here as well.

If you get a chance to do this, there's a typo that can be fixed as well: `arcihve` under `0.0.9`.

Reply:
@myii, I have the idea to update the formula to follow `template` (see #53) so all this would be fixed then? I can reverse it in the meantime, not big deal and I have to work on your other suggestions anyway :smile:

Reply:
Done